src/mosquito/base.cr

```diff
@@ -46,5 +49,20 @@ module Mosquito
     def self.log(*messages)
       logger.log(Logger::Severity::INFO, messages.join(" "))
     end
+
+    def self.idle_wait : Float64
+      @@idle_wait
+    end
+
+    def self.idle_wait=(@@idle_wait : Float64)
+    end
```

Comment:
```suggestion
  class_property idle_wait : Float64 = 0.1
```

https://github.com/robacarp/mosquito/pull/44/files#diff-ab966f0737644cd4c1910f0bc888cf57R11 could be removed as well.